source/manage/client-proxysql-metrics.rst

```diff
@@ -35,12 +35,14 @@ The output of this command may look as follows:
 
 Beside positional arguments shown above you can specify service name and
 service address with the following flags: ``--service-name``, ``--host`` (the
-hostname or IP address of the service), and ``--port`` (the port number of the
-service). If both flag and positional argument are present, flag gains higher
-priority. Here is the previous example modified to use these flags::
+hostname or IP address of the service), and either ``--port`` (the port number of the
```

Comment:
I guess it should here something like: host with port or socket.

Reply:
I adjusted the sentence and example to show host/port vs socket variants. If you think it needs to be clearer, I will expand the description but it would probably go somewhere else, as this host/port vs socket choices apply to all connectors.